src/main/scala/beam/sim/vehicles/VehiclesAdjustment.scala

```diff
@@ -28,12 +28,17 @@ trait VehiclesAdjustment extends LazyLogging {
 }
 
 object VehiclesAdjustment {
-  val DEFAULT_ADJUSTMENT = "DEFAULT_ADJUSTMENT"
-  val UNIFORM_ADJUSTMENT = "UNIFORM_ADJUSTMENT"
+  val DEFAULT_ADJUSTMENT = "DEFAULT"
```

Comment:
what is difference between DEFAULT and UNIFORM?

Reply:
Not totally sure--both were there before so I just kept them. I'll merge them to simplify.